common-pipeline/operators/uihints-v3-schema.json

```diff
@@ -662,6 +669,65 @@
           }
         }
       ]
+    },
+    "button_definition": {
+      "description": "Button to be displayed after the table's search bar",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Button identifier used to idenify the button in the callback function",
+          "type": "string"
+        },
+        "label": {
+          "description": "Button label to display. If an icon is specified as well, the icon will be shown to the right of the label.",
+          "type": "object",
+          "$ref": "#/definitions/resource_definition"
+        },
+        "description": {
+          "description": "Tooltip text to display when the button is hovered",
+          "type": "object",
+          "$ref": "#/definitions/resource_definition"
+        },
+        "icon": {
+          "description": "URL to image to display",
+          "type": "string"
+        },
+        "carbon_icon": {
+          "description": "Host provided name of Carbon icon to display in the button's 'renderIcon'. A callback function is required for the host application to return the actual imported icon from @carbon/icons-react library",
+          "type": "string"
+        },
+        "divider": {
+          "description": "Display a divider before or after this button",
+          "type": "string",
+          "enum": [
+            "before",
+            "after"
+          ],
+          "default": "after"
+        }
+      },
+      "anyOf": [
+        {
+          "required": [
+            "id",
+            "label"
+          ]
+        },
+        {
+          "required": [
+            "id",
+            "icon",
+            "description"
+          ]
+        },
+        {
+          "required": [
+            "id",
+            "carbon_icon",
+            "description"
```

Comment:
Carbon icons require a description to be displayed in the tooltip. Making this required for any icon only button to stay consistent